testing/prow/benchmarking.sh

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env bash
+
+set -x
+set -o pipefail
+set -o errexit
+set -o nounset
+
+PSAP_SECRET_PATH=/var/run/psap-entitlement-secret
+EPOCHS=3
+THRESHOLD=0.05
+MINSR=150.0
+MAXDR=30
+
+assess_benchmark_stats () {
+    artifact_dir=${ARTIFACT_DIR:-"/tmp/ci-artifacts_$(date +%Y%m%d)"}
+    sample_rate=$(cat "${artifact_dir}/benchmarking_run_ssd_sample_rate.log")
+    bench_duration=$(cat "${artifact_dir}/benchmarking_run_ssd_bench_duration.log")
```

Comment:
mind here for the unit, I guess you can add `| cut -d" " -f1` to remove the unit